services/API-service/package.json

```diff
@@ -78,5 +78,13 @@
     "tsconfig-paths": "^3.9.0",
     "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.3/xlsx-0.20.3.tgz",
     "yargs": "^17.0.0"
+  },
+  "lint-staged": {
+    "**/*.{js,jsx,ts,tsx,html,css,scss,json,md}": [
```

Comment:
You are deviating from what is laid out in the root directory here (eg. using prettier instead of eslint for js files), but maybe this is an intentional override.

Reply:
it was not, I'll adjust, thanks!